Index: vector.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vector.h b/vector.h
new file mode 100644
--- /dev/null	(date 1732112297352)
+++ b/vector.h	(date 1732112297352)
@@ -0,0 +1,8 @@
+//
+// Created by Mali Zhu on 2024/11/20.
+//
+
+#ifndef PROGETTOINTERMEDIO_VECTOR_H
+#define PROGETTOINTERMEDIO_VECTOR_H
+
+#endif //PROGETTOINTERMEDIO_VECTOR_H
